firefox-ios/firefox-ios-tests/Tests/ClientTests/FxAWebViewModelTests.swift

```diff
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+@testable import Client
+import XCTest
+import PDFKit
+
+// swiftlint:disable force_try
```

Comment:
@dicarobinho, do you really need this as my guess is its only for 

https://github.com/mozilla-mobile/firefox-ios/pull/20215/files#diff-bf65ed39518f585b6ff439e5e20ad51766a498496762890aebc5ade1950534eeR114-R115

Reply:
And other cases, if we can improve the force try then that would be amazing

Reply:
Fixed them.